src/improvements/SuperchainAddressRegistry.sol

```diff
@@ -299,8 +300,7 @@ contract SuperchainAddressRegistry is StdChains {
         address anchorStateRegistryProxy = getAnchorStateRegistryProxy(permissionedDisputeGame);
         saveAddress("AnchorStateRegistryProxy", chain, anchorStateRegistryProxy);
 
-        // Not retreiving delayed WETH proxy because 'n' exist based on the number of GameTypes.
-        // We will leave these addresses for the task developer to retrieve.
+        saveAddress("PermissionedWETH", chain, getDelayedWETHProxy(permissionedDisputeGame));
```

Comment:
Previously we said,
> // Not retreiving delayed WETH proxy because 'n' exist based on the number of GameTypes.
> // We will leave these addresses for the task developer to retrieve.

but this proved difficult for me as the task developer of OPCMUpgradeV200. 

I had two options:
1. Require task dev to define the address as an input in `config.toml`. This gets super messy because you have to then add these addresses to the `_taskStorageWrites` function. When you have > 1 chain you have to start coming up with different names for each contract. 
2. Get both Permissioned and Permissionless WETH in discovery and have a clean `_taskStorageWrites` function in the template that doesn't have chain specific names.